app/assets/stylesheets/content/_simple_filters.sass

```diff
@@ -91,6 +91,24 @@ $filters--border-color: $gray !default
     @include text-shortener
     margin: auto 0
 
+fieldset#date-range
+  p
```

Comment:
There is no `<p>` within that specific fieldset. So we can remove that rule.

Reply:
It is removed now.